Makefile

```diff
@@ -1,24 +1,34 @@
 PWD := $(shell pwd)
 
-get-deps:
-	go get github.com/go-resty/resty
+# API Spec branch name
+BRANCH := master
 
 generate: clean
+	# pull spec
+	curl -s https://raw.githubusercontent.com/giantswarm/api-spec/$(BRANCH)/spec.yaml > api-spec/spec.yaml
+	curl -s https://raw.githubusercontent.com/giantswarm/api-spec/$(BRANCH)/responses.yaml > api-spec/responses.yaml
+	curl -s https://raw.githubusercontent.com/giantswarm/api-spec/$(BRANCH)/parameters.yaml > api-spec/parameters.yaml
+	curl -s https://raw.githubusercontent.com/giantswarm/api-spec/$(BRANCH)/definitions.yaml > api-spec/definitions.yaml
 	docker run --rm -it \
-		-v ${PWD}:/swagger-api/out \
-		-v ${PWD}/api-spec:/swagger-api/yaml \
-		jimschubert/swagger-codegen-cli:2.2.3 generate \
-		--input-spec /swagger-api/yaml/oai-spec.yaml \
-		--lang go \
-		--config /swagger-api/out/swagger-codegen-conf.json \
-		--output /swagger-api/out
-	gofmt -s -l -w .
+	  -e GOPATH=${HOME}/go:/go \
+		-v ${HOME}:${HOME} \
```

Comment:
Just being picky here, does it works without  `GOPATH` definition and volume mounting ?

Reply:
I'm pretty sure these have to be set. It's been end of May when I figured this out, so I can't tell exactly. But I'm certain I put that there for a reason.

Reply:
In fact there is some points to correct
* create the `api-spec` directory
* `GOPATH` env var is not needed, its already set to `/go`
* do not mount `${HOME}`, so we do not mess with users home directory
* mount `${PWD}` to `/go/src/anything`
* use `/go/src/anything/api-spec` as working directory

Reply:
All right, changed